daemon/container_operations_windows.go

```diff
@@ -58,3 +43,7 @@ func isLinkable(child *container.Container) bool {
 func enableIPOnPredefinedNetwork() bool {
 	return true
 }
+
+func (daemon *Daemon) supportsHotAdd() bool {
```

Comment:
- That's quite a vague name: hot adding _what_?
- Does it need to be a function instead of a platform-specific compile-time constant?